recipes/r-stars/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Already talked with upstream, this license should be in the next release tarball: https://github.com/r-spatial/stars/issues/195